api-put-object.go

```diff
@@ -77,8 +78,11 @@ func (opts PutObjectOptions) Header() (header http.Header) {
 		header[amzHeaderKey] = []string{opts.EncryptMaterials.GetKey()}
 		header[amzHeaderMatDesc] = []string{opts.EncryptMaterials.GetDesc()}
 	}
+	if opts.StorageClass != "" {
+		header["X-Amz-Storage-Class"] = []string{opts.StorageClass}
```

Comment:
You might have to add a const for this string.

Reply:
sure will do